### ISSUE TITLE
Governance process

### ISSUE THREAD

**Issue by rmontuoro** [ESCOMP/PhysicalConstantsDictionary#9]
A shared governance process needs to be established to take charge of:

- reviewing and accepting/rejecting new physical constant sets submitted by the modeling community
- preparing official revisions of the physical constant dictionary, which include accepted constant sets from the community
- releasing and advertising each official revision of the physical constant dictionary

The governance process may include a steering committee/board with elected representatives of major research and modeling centers.

**Comment by gold2718** [comment]
I suggest a lightweight governance model which will accept membership from any organization (e.g., NOAA, NCAR, NRL) which formally commits resources to the development and maintenance of the CPCD. One example we could follow is the CCPP Framework governance (https://github.com/NCAR/ccpp-framework/wiki/CCPP-Framework-Governance) which follows this model.

**Comment by rmontuoro** [comment]
I agree with @gold2718 that a lightweight governance model with would be suitable for CPCD.

I suggest creating a governance board whose members are appointed by ESPC stakeholders and community representatives. Each participating institution may appoint one science and one technical representative (they may also be the same person) to supervise and contribute to the CPCD content and development process.

Information about the governance body, process, and membership, along with guidelines describing how to contribute and use CPCD should be posted to a public web page (_e.g._ this project's Wiki).
